book/09-git-and-other-scms/sections/import-bzr.asc

```diff
@@ -0,0 +1,154 @@
+$ git init
+----
+
+Now, you can simply export your Bazaar repository and convert it into a Git repository using the following command:
+[source,console]
+----
+$ bzr fast-export --plain . | git fast-import
+----
+
+You may get the following error message:
+[source,console]
+----
+bzr: ERROR: Unable to import library "fastimport": bzr-fastimport requires the fastimport python module
+----
+
+This message is displayed because the fastimport python module that the `bzr fast-export` command needs is missing.
+Install it from your package manager if you are under GNU/Linux or download it at address https://pypi.python.org/pypi/fastimport/[] if you are under Windows.
+For example, with Fedora, you would do the following:
+
+[source,console]
+----
+$ sudo dnf install python-fastimport
+----
+
+Now you can export your Bazaar repository to the Git repository.
+
+Depending on the size of the project, your Git repository is built in a lapse from a few seconds to a few minutes.
+
+At this point, if you type `git status`, you'll see that the tracked files are now marked as removed for the next commit.
+Here is an example:
+[source,console]
+----
+$ git status
+On master branch
+Changes that will be validated:
+  (use "git reset HEAD <fichier>..." to unstage)
+
+        removed :        .bzrignore
+        removed :        file.txt
+
+Untracked files:
+  (use "git add <fichier>..." to include in what will be validated)
+
+        .bzr/
+        .bzrignore
+        file.txt
+----
+
+You can restore the repository in a correct state very simply with the `git reset` command:
+[source,console]
+----
+$ git reset HEAD .
+----
+
+Now let us have a look at the files to ignore.
+As .bzrignore's format is completely compatible with .gitignore's format, the simplest is to rename your .bzrignore file:
+[source,console]
+----
+$ git mv .bzrignore .gitignore
+----
+
+Let us check your repository's status:
+[source,console]
+----
+$ git status
+On master branch
+Changes that will be validated :
+  (use "git reset HEAD <fichier>..." to unstage)
+
+        renamed :         .bzrignore -> .gitignore
+
+Untracked files:
+  (use "git add <fichier>..." to include in what will be validated)
+
+        .bzr/
+
+----
+
+Then you have to create a commit that contains those changes for the migration:
+[source,console]
+----
+$ git commit -am 'Migration from Bazaar to Git'
+----
+
+That's all!
+Now you can push the repository onto its new home server:
+[source,console]
+----
+$ git remote add origin git@my-git-server:mygitrepository.git
+$ git push origin --all
+$ git push origin --tags
+----
+
+===== Case of a project with a main branch and a working branch
+
+You can also import a Bazaar repository that contains branches.
+Let us suppose that you have two branches: one represents the main branch (myProject.trunk), the other one is the working branch (myProject.work).
+[source,console]
+----
+$ ls
+myProject.trunk myProject.work
+----
+
+Create the Git repository and go into it:
```

Comment:
Why ? Isn't it clear with the verb to go ? Even if it was not the case, the command is given three lines later.